ios/FluentUI.Demo/FluentUI.Demo/Resources/Assets.xcassets/Button/Placeholder_24.imageset/Contents.json

```diff
@@ -0,0 +1,16 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "placeholder_24.pdf"
```

Comment:
if this is coming from fluent icon repo, do you mind keeping the same name so we can easily update it later?

Reply:
That is the name from the fluent icon repo.

Reply:
This is the name that's being used in figma. Is that the right name? I'm not sure what's the fluent icon repo.

Reply:
Resolving assuming that Fluent icons == what's on Figma

Reply:
Ok, I copied over the resources directly from the fluent icon repo with the same name. Thanks.